packages/big-design-theme/src/system/colors.ts

```diff
@@ -49,6 +52,9 @@ export interface Colors {
 }
 
 export const colors: Colors = {
+  inherit: 'inherit',
+  transparent: 'transparent',
```

Comment:
🍹 What do you think about adding `currentColor` as well?

Reply:
Might as well